practice/src/scripts/constants/notify.ts

```diff
@@ -0,0 +1,60 @@
+import { Notify } from "./types";
+
+export const messageNotify: Array<{
```

Comment:
create a object for status
a object for message.
if create messageNotify it will duplicate message and type in a object